tiledb/api/c_api/attribute/attribute_api_external.h

```diff
@@ -305,7 +305,7 @@ TILEDB_EXPORT int32_t tiledb_attribute_get_cell_size(
  * @param out The output.
  * @return `TILEDB_OK` for success and `TILEDB_ERR` for error./
  */
-TILEDB_EXPORT int32_t tiledb_attribute_dump(
+TILEDB_DEPRECATED_EXPORT int32_t tiledb_attribute_dump(
```

Comment:
This should be surrounded in an `#ifndef TILEDB_REMOVE_DEPRECATIONS`.

Reply:
Moved all of them to `tiledb_deprecated.h`

Reply:
Even better, I forgot about it.

Reply:
> Moved all of them to tiledb_deprecated.h

How is that? That file isn't even in this PR.

Reply:
Maybe he didn't push yet.